torch/csrc/jit/script/builtin_functions.cpp

```diff
@@ -31,11 +31,17 @@ def div(a : ${Scalar}, b : Tensor) -> Tensor:
 auto python_builtins_source = R"SCRIPT(
 def warn(string: str):
   print(string)
+def max_pool1d(input, kernel_size, stride=None, padding=0, dilation=1, ceil_mode=False, return_indices=False):
```

Comment:
`Template[bool, True]` is redundant here, right? because a default value is also set.

Reply:
Why are these implemented in builtin_functions.cpp rather with the normal standard library?